### DISCUSSION TITLE
Is it possible to scale when threshold drops below a value?

### DISCUSSION THREAD

**Discussion by jesumyip** [kedacore/keda#5247] [Q&A / Need Help]
I have a situation where a particular task is run on a schedule (once every 2seconds). This tells me if the process running the task is still alive.  I export this to Newrelic

If the process is overwhelmed, that task may only run at the next 15second window instead of 2 second. 

I want keda to autoscale my pods if the NRQL query returns a value of less than 30 for a `SELECT count`. Most of the examples I have seen is for keda to trigger autoscaling when the value is MORE than 30.

**Comment by jesumyip** [comment]
https://keda.sh/docs/2.12/concepts/scaling-deployments/#scaling-modifiers-experimental

It looks like the `ternary operator` would be what I require?

**Reply by zroubalik** [reply]
Yeah, definitely give it a try and please report back if you were successful.

**Comment by jesumyip** [comment]
Follow-up question. Is there a way I can get the current number of replicas to be used in calculations? Example:

1. I have a readinessProbe that runs every 5 seconds.
2. Each replica therefore generate 12 readinessProbe calls (assuming all are successful) per minute.
3. In NewRelic, I can see 120 probes generated every 10minutes  if I have 1 replica.
4. If Keda scales this deployment up to 2 replicas, that NRQL then changes to 240 probes per 10minutes.
How can I update automatically the Keda ScaledObject to now watch for 240 probes instead of 120 probes per 10 minutes?
